template/data/fr/pages.json

```diff
@@ -0,0 +1,308 @@
+{
+  "pagesFR": [
+    {
+      "id": 5,
+      "slug": "french",
```

Comment:
I think this should be an empty string, otherwise it won't be the root of the /fr website

But I can see that it's breaking the unique constraint in Strapi. So I really don't don't know what the fix would be right now. I'll come back to it tomorrow

Reply:
Hey thanks for taking a look.  Yeah I agree will need to find workaround for that.  

I should have mentioned the /fr seed data is just a lot of nonsense to check if everything is working.

Reply:
Yes I figured, nice song btw